application/cmd/requester/utils_test.go

```diff
@@ -0,0 +1,130 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/insolar/insolar/insolar/secrets"
+)
+
+const (
+	binLocation                string = "./../../../bin/requester"
+	getSeedResponse            string = `{"jsonrpc":"2.0","result":{"seed":"s2pMpKZwIeqOaHbVkWQQFnzMPqotBKwEnBElZCTmz3w=","traceID":"c648d2ee1e414ce7f12f266cef47eeac"},"id":0}`
+	memberCreateResponse       string = `{"jsonrpc":"2.0","result":{"callResult":{"reference":"insolar:1AhiuscmUtUs4gqtSQFcKj-PV5Z1hp3GVGG4SPtdxkvs"},"requestReference":"insolar:1AhiusW4AvZu_0tdLO0Vq_cH9Hoc62vU4W0vJApydA1I.record","traceID":"f441b82fb75cc744595524353140e0b2"},"id":8674665223082153551}`
+	createMemberRequestExample string = `{
+  "jsonrpc": "2.0",
+  "method": "contract.call",
+  "id": 1,
+  "params": {
+    "seed": "fhDEwRRbSnYnbMnALKMh8gXdzaSvRv/nfsGC9S7kqik=",
+    "callSite": "member.create",
+    "publicKey": "-----BEGIN PUBLIC KEY-----\\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEMSbA4KvO/jlwY+8WFDEdwhCLlsEC\\nF3/GYvu9iTWHwCctx1wTbGGjNLY03EjXyYxaf8coNbSbZeu+jXcWeMHG0A==\\n-----END PUBLIC KEY-----"
+  }
+}`
+)
+
+func handlers() http.Handler {
+	r := http.NewServeMux()
+	r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		localVarBody, err := ioutil.ReadAll(r.Body)
+		_ = r.Body.Close()
+		w.WriteHeader(http.StatusOK)
+		if err == nil {
+			body := strings.ToLower(string(localVarBody))
+			if strings.Contains(body, "node.getseed") {
+				if _, err := fmt.Fprint(w, getSeedResponse); err != nil {
+					log.Println(err.Error())
+				}
+			} else if strings.Contains(body, "member.create") {
+				if _, err := fmt.Fprint(w, memberCreateResponse); err != nil {
+					log.Println(err.Error())
+				}
+			}
+		} else {
+			if _, err := fmt.Fprint(w, "{\"requestReference\" :\"sddasacs\"'}"); err != nil {
+				log.Println(err.Error())
+			}
+		}
+	})
+	return r
+}
+
+func runCmd(args ...string) (string, error) {
+	cmd := exec.Command(binLocation, args...)
+	out, err := cmd.CombinedOutput()
+	return string(out), err
+}
+
+func getArgs(key, url, requestParams string, shouldPrivateKey, shouldSeed bool) []string {
+	cmd := make([]string, 5)
+	cmd[0] = fmt.Sprintf("-k=%v", key)
+	cmd[1] = fmt.Sprintf("-u=%v", url)
+	cmd[2] = fmt.Sprintf("-r=%v", requestParams)
+	cmd[3] = fmt.Sprintf("-p=%v", shouldPrivateKey)
+	cmd[4] = fmt.Sprintf("-s=%v", shouldSeed)
+	return cmd
+}
+
+func writePrivateKeyToFile(pathToFile string) {
```

Comment:
put asserts to `testdata` dir, don't generate them there is no reason for it here

Reply:
I still think it's not a test job generate asserts in this case.

Reply:
it's kind of hidden dependency antipattern. I really think that using the created files is a bad idea. 
I'll use if you want. No problem

Reply:
I disagree. test asserts is well known and good practice, and not antipattern here